api/src/paths/project/{projectId}/survey/{surveyId}/critters/{critterId}/deployments.ts

```diff
@@ -100,8 +100,8 @@ POST.apiDoc = {
       content: {
         'application/json': {
           schema: {
-            title: 'Number of rows affected',
-            type: 'number'
+            title: 'Deployment response object',
+            type: 'object'
```

Comment:
nit: specify that this is an object with 1 field "message"

Same for the other 201 response

Reply:
Can make it required too, since its always provided.